mastering-plone/features.rst

```diff
@@ -42,13 +42,19 @@ Some commands you will use rather often are::
 
     A standard installation listens on port 8080, so lets have a look at our Zope site by visiting http://localhost:8080
 
+	.. image:: _static/features_plone_running.png
+		:scale: 50 %
```

Comment:
The indention of the options is a little much. It is still valid rst I think but I think 4 spaces should be enough.